packages/core/src/components/axes/two-dimensional-axes.ts

```diff
@@ -111,5 +117,27 @@ export class TwoDimensionalAxes extends Component {
 
 			this.render(true);
 		}
+
+	}
+
+	addAxisThresholds(axisPosition) {
+		const axesOptions = Tools.getProperty(this.model.getOptions(), "axes", axisPosition);
+		const { thresholds } = axesOptions;
+
+		if (thresholds) {
```

Comment:
can thresholds move around in data updates? will they render, de-render or re-render properly?

Reply:
They are rendered, de-render, and re-render when `TwoDimensionalAxes` does, in the same way as axes (TwoDimensionalAxes children) do, 
Should they behave differently?

Do you think it would be better to add a `transition` to them?

Reply:
if it's not too distracting I think a transition might make for a smoother experience